sdk/src/vm.rs

```diff
@@ -9,14 +10,31 @@ pub const NO_DATA_BLOCK_ID: u32 = 0;
 
 /// Abort execution.
 pub fn abort(code: u32, message: Option<&str>) -> ! {
+    exit(code, message, None)
```

Comment:
Check that the code is non-zero, and convert it into something else if it is (assertion failed?).

Reply:
done.